sdk/groups_report.go

```diff
@@ -0,0 +1,34 @@
+package sdk
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	apiError "github.com/ONSdigital/dp-identity-api/sdk/errors"
+)
+
+type GroupsReportResponse struct {
```

Comment:
I'd possibly either rename this type or change it's structure. The name implies that this type is the entire response. Maybe…
```suggestion
type GroupsReportResponse []struct {
```
… or more completely…
```suggestion
type GroupsReportResponse []GroupsReportItem
type GroupsReportItem struct {
```